src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -323,10 +323,10 @@ public PlatformDetectorResult Detect(RepositoryContext context)
             var frameworksObj = nodePlatformDetectorResult.Frameworks;
             if (frameworksObj != null && frameworksObj.Any())
             {
-                string[] framworks = frameworksObj.Select(p => p.Framework).ToArray();
-                string frameworks = string.Join(",", framworks);
+                string frameworks = string.Join(",", frameworksObj.Select(p => p.Framework).ToArray());
                 manifestFileProperties[ManifestFilePropertyKeys.Frameworks] = frameworks;
                 this.logger.LogInformation($"Detected the following framework(s): {frameworks}");
+                Console.WriteLine($"Detected the following framework(s): {frameworks}");
```

Comment:
nit: How about other platforms like python, php and golang, should we update those as well?

Reply:
These were the only specified frameworks that were specified by the static websites team